lms/static/scripts/frontend_apps/components/ErrorDisplay.tsx

```diff
@@ -98,6 +98,11 @@ export type ErrorDisplayProps = {
    * response.
    */
   error: ErrorLike;
+
+  supportLink?: boolean;
+  /**
+   * Wether or not to show the standard link to our support page
+   */
```

Comment:
```suggestion
  /**
   * Wether or not to show the standard link to our support page
   */
  supportLink?: boolean;
```

JSDocs usually go before the symbol they describe 😅